packages/storybook/stories/CallComposite/snippets/CustomDataModelExampleContainer.snippet.tsx

```diff
@@ -69,13 +70,29 @@ export const CustomDataModelExampleContainer = (props: ContainerProps): JSX.Elem
       });
     });
 
+  // Custom Menu Item Callback for Participant List
+  const onFetchParticipantMenuItems: ParticipantMenuItemsCallback = (participantId, userId, defaultMenuItems) => {
```

Comment:
update params to be typed here
```suggestion
  const onFetchParticipantMenuItems: ParticipantMenuItemsCallback = (participantId: string, userId: string, defaultMenuItems: something[]) => {
```

Reply:
Seems redundant given that we have already provided a type to `onFetchParticipantMenuItems`